images.yaml

```diff
@@ -583,6 +583,24 @@
 - name: quay.io/calico/kube-controllers
   patterns:
   - pattern: '>= v3.9.5'
+- name: quay.io/calico/kube-controllers
+  overrideRepoName: calico-crd-installer
+  patterns:
+  - pattern: '>= v3.16.0'
+    customImages:
+    - dockerfileOptions:
+      - |
+        FROM quay.io/giantswarm/crd-installer:0.0.0-add8f5797322a6e44489740b17a59c0a64ae36e7 AS installer
```

Comment:
Will replace with 0.0.1 once this PR https://github.com/giantswarm/crd-installer/pull/1 is released.